src/components/Dialog/index.tsx

```diff
@@ -97,110 +92,96 @@ const Dialog = ({
+                    <div>
+                      <div className="padding--right--xl type--head4">
+                        {title ? <span id="a11y-dialog-title">{title}</span> : '\u00A0'}{' '}
+                        {/* There always needs to be something (even a space) in the header for display reasons */}
+                      </div>
+                      {subTitle}
                     </div>
-                    {subTitle}
+                    {onDismiss && (
+                      <DialogUi.Trigger asChild>
+                        <div className="dialog-icon">
+                          <IconButton Icon={DenyIcon} aria-label="Close" label="Close" />
+                        </div>
+                      </DialogUi.Trigger>
+                    )}
                   </div>
-                  {onDismiss && (
-                    <div className="dialog-icon">
-                      <IconButton
-                        Icon={DenyIcon}
-                        onClick={onDismiss}
-                        aria-label="Close"
-                        label="Close"
-                      />
-                    </div>
-                  )}
                 </div>
+              </>
+            )}
+            {content && (
+              <div className="dialog-content" ref={contentEl}>
+                <div className="padding--all bgColor--white">{content}</div>
               </div>
-            </React.Fragment>
-          )}
-          {content && (
-            <div className="dialog-content" ref={contentEl}>
-              <div className="padding--all bgColor--white">{content}</div>
-            </div>
-          )}
-          {footerContent && (
-            <div className="dialog-footer">
-              <div
-                className={cc([
-                  {
-                    'border--top': alwaysShowBorder || isOverflowing,
-                  },
-                  'bgColor--white',
-                  'padding--all',
-                ])}
-              >
-                {footerContent}
+            )}
+            {footerContent && (
+              <div className="dialog-footer">
+                <div
+                  className={cc([
+                    {
+                      'border--top': alwaysShowBorder || isOverflowing,
+                    },
+                    'bgColor--white',
+                    'padding--all',
+                  ])}
+                >
+                  {footerContent}
+                </div>
               </div>
-            </div>
-          )}
-        </div>
+            )}
+          </div>
+        </DialogUi.Content>
       </div>
-    </div>
+    </DialogUi.Root>
   );
 
   const transitionNode = (
     <CSSTransition timeout={200} in={isOpen} classNames="dialog" unmountOnExit>
-      <React.Fragment>
-        {disableFocusTrap ? dialogNode : <FocusTrap>{dialogNode}</FocusTrap>}
-      </React.Fragment>
+      <>{disableFocusTrap ? dialogNode : <FocusTrap>{dialogNode}</FocusTrap>}</>
```

Comment:
Do we need this?

Reply:
i think so - we sometimes need to disableFocusTrap

Reply:
i confirmed that the prop is being used on cbi-site

Reply:
right. But I guess my question, is that Radix focus traps by default. Even if that prop is being used on `cbi-site`, the focus is still going to be trapped for that instance
<img width="529" alt="Screen Shot 2022-03-18 at 4 40 33 PM" src="https://user-images.githubusercontent.com/708820/159086470-2d7d572c-016d-4d19-bb08-5eacd0138636.png">

Reply:
ah, right good call. I'll remove it

Reply:
the disableFocusTrap prop is now being used to toggle radix's modal prop to conditionally focus/disableFocus